docker-compose.lando-ui.yml

```diff
@@ -10,7 +10,7 @@ services:
       dockerfile: ./docker/Dockerfile-dev
     image: suite_lando-ui
     environment:
-      VERSION_PATH: /version.json
+      VERSION_PATH: /config/version.json
```

Comment:
We are creating `version.json` at the root directory in `Dockerfile-dev` (and `Dockerfile-prod`). So when we start this container, this version path is actually referencing an incorrect path (i.e. not matching with the actual version file path in the container). This is a little confusing.

This also seems to cause a failure in `lando-ui`:
```
Traceback (most recent call last):
  File "/usr/local/bin/lando-ui-dev", line 11, in <module>
    load_entry_point('lando-ui', 'console_scripts', 'lando-ui-dev')()
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/app/landoui/app.py", line 223, in run_dev_server
    str2bool(debug)
  File "/app/landoui/app.py", line 67, in create_app
    version_info = json.load(open(version_path))
FileNotFoundError: [Errno 2] No such file or directory: '/config/version.json'```

Reply:
Ah, good eye, I kept a close eye on `lando-api`, but didn't give the same scrutiny to `lando-ui` :+1: